wizard-common/src/main/java/org/cobbzilla/wizard/validation/Validator.java

```diff
@@ -42,7 +44,8 @@ public Validator(ValidatorFactory factory) {
     public ValidationResult validate(Object o, Class<?>... groups) {
         final List<ConstraintViolation> violations = new ArrayList<>();
         if (o == null) {
-            violations.add(EmptyRequestEntityConstraintViolation.INSTANCE);
+            violations.add(forReturnValueValidation("err.requestEntity.null", null, null, null, null, null, null, null,
```

Comment:
Can the `forReturnValueValidation` be made into a constant? Validator is used a lot and it would be nice to avoid an additional memory allocation every time there are no violations.